src/main/java/org/threeten/extra/chrono/FrenchRepublicDate.java

```diff
@@ -199,19 +200,21 @@ ValueRange rangeAlignedWeekOfMonth() {
 
     @Override
     public ValueRange range(TemporalField field) {
-        if (field.equals(WeekFields.ISO.dayOfWeek()))
+        if (DAY_OF_WEEK.equals(field)) {
             return DOW_RANGE;
+        }
```

Comment:
DAY_OF_WEEK is dealt with in the chronology via super class. So I'll remove this entirely